code/modules/paperwork/contract.dm

```diff
@@ -262,23 +262,23 @@
 	to_chat(user, "<span class='notice'>A profound emptiness washes over you as you lose ownership of your soul.</span>")
 	to_chat(user, "<span class='boldnotice'>This does NOT make you an antagonist if you were not already.</span>")
 	SSblackbox.record_feedback("tally", "infernal contract", 1, contractType)
-	return TRUE
+	return FALSE
 
 /obj/item/paper/contract/infernal/proc/signIncorrectly(mob/living/carbon/human/user = target.current, blood = FALSE)
-	signed = 1
+	signed = TRUE
 	update_text("your name", blood)
 
 /obj/item/paper/contract/infernal/power/fulfillContract(mob/living/carbon/human/user = target.current, blood = FALSE)
 	if(!user.dna)
-		return -1
+		return TRUE
```

Comment:
```suggestion
		return -1
```
this needs to stay negative because it's checked later on in [`attempt_signature`](https://github.com/tgstation/tgstation/blob/a28b24f149702527f3eb22f5c686f06c836f2f99/code/modules/paperwork/contract.dm#L214)

Reply:
That is so stupid ill fix that

Reply:
Changed how fullfilContract works, adressed